src/io/import/processors/handleConfig.ts

```diff
@@ -0,0 +1,56 @@
+import { z } from 'zod';
+
+import { ImportHandler } from '@/src/io/import/common';
+import { useRectangleStore } from '@/src/store/tools/rectangles';
+import { useRulerStore } from '@/src/store/tools/rulers';
+
+const Color = z.string();
+
+const Label = z.object({
+  color: Color,
+});
+
+const RectangleLabel = z.intersection(
+  Label,
+  z.object({
+    fillColor: Color,
+  })
+);
+
+const Config = z.object({
+  labels: z.record(Label).or(z.null()),
+  rectangleLabels: z.record(RectangleLabel).or(z.null()),
+});
+
+const readConfigFile = async (configFile: File) => {
+  const decoder = new TextDecoder();
+  const ab = await configFile.arrayBuffer();
+  const text = decoder.decode(new Uint8Array(ab));
+  return Config.parse(JSON.parse(text));
+};
+
+const applyConfig = (manifest: z.infer<typeof Config>) => {
+  useRectangleStore().setLabels(manifest.rectangleLabels);
+  useRulerStore().setLabels(manifest.labels);
```

Comment:
Suggestion: `labels` should be `rulerLabels`. Unless this is meant to be a generic/abstract "labels" definition?

Reply:
I was thinking it could be the generic fallback "labels" but I did not implement. Now I did like this:
```
const applyConfig = (manifest: z.infer<typeof Config>) => {
  useRulerStore().setLabels(manifest.labels);
  useRectangleStore().setLabels(manifest.rectangleLabels ?? manifest.labels);
};
```

Reply:
In that case, I would also suggest allowing for a `rulerLabels` as well, since `labels` is the default fallback and each other tool can have their own labels.

Reply:
Added the `rulerLabels` property for the config.